core/src/main/scala/akka/persistence/jdbc/schema/javadsl/SchemaUtils.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (C) 2014 - 2019 Dennis Vriend <https://github.com/dnvriend>
+ * Copyright (C) 2019 - 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.jdbc.schema.javadsl
+
+import java.util.concurrent.CompletionStage
+
+import akka.Done
+import akka.actor.ClassicActorSystemProvider
+import akka.persistence.jdbc.schema.scaladsl
+import scala.compat.java8.FutureConverters._
+import scala.concurrent.Future
+
+import akka.dispatch.Dispatchers
+import akka.persistence.jdbc.db.SlickDatabase
+import akka.persistence.jdbc.db.SlickExtension
+import slick.jdbc.JdbcBackend.Database
+
+object SchemaUtils {
```

Comment:
```suggestion
object TestingUtils {
```

Reply:
Not sure if we should to it. Isn't `testkit.SchemaUtils` good enough?

Reply:
`testkit` package is enough in my opinion